export const throwReferenceError = (message) => {
  throw new ReferenceError(message);
};
